packages/zoe/src/contractFacet/zcfMint.js

```diff
@@ -1,169 +1,163 @@
-import { E } from '@endo/eventual-send';
 import { AmountMath } from '@agoric/ertp';
-import {
-  provideDurableSetStore,
-  makeScalarBigMapStore,
-  prepareSingleton,
-  prepareExo,
-  M,
-} from '@agoric/vat-data';
+import { M, prepareExoClass } from '@agoric/vat-data';
+import { E } from '@endo/eventual-send';
 
 import { coerceAmountKeywordRecord } from '../cleanProposal.js';
-import { makeIssuerRecord } from '../issuerRecord.js';
+import { assertFullIssuerRecord, makeIssuerRecord } from '../issuerRecord.js';
 import { addToAllocation, subtractFromAllocation } from './allocationMath.js';
 
 import '../internal-types.js';
+import {
+  AmountKeywordRecordShape,
+  IssuerRecordShape,
+  SeatShape,
+} from '../typeGuards.js';
 import './internal-types.js';
 import './types.js';
 
 const { Fail } = assert;
 
-export const makeZCFMintFactory = (
-  /** @type {import('@agoric/vat-data').Baggage} */ zcfBaggage,
-  /** @type {{ (keyword: string, issuerRecord: IssuerRecord): void }} */ recordIssuer,
-  /** @type {GetAssetKindByBrand} */ getAssetKindByBrand,
-  /** @type { (exit?: undefined) => { zcfSeat: any; userSeat: Promise<UserSeat> }} */ makeEmptySeatKit,
-  /** @type {ZcfMintReallocator} */ reallocator,
+/**
+ *
+ * @param {import('@agoric/vat-data').Baggage} zcfBaggage
+ * @param {{ (keyword: string, issuerRecord: IssuerRecord): void }} recordIssuer
+ * @param {GetAssetKindByBrand} getAssetKindByBrand
+ * @param {(exit?: undefined) => { zcfSeat: any; userSeat: Promise<UserSeat> }} makeEmptySeatKit
+ * @param {ZcfMintReallocator} reallocator
+ */
+export const prepareDurableZcfMint = (
```

Comment:
When a function is already named `prepare*` and takes a baggage as first argument, the "Durable" part of the name is implied, and so redundant. My naming habit is to say "durable" explicitly only when the name does not start with "prepare".

```suggestion
export const prepareZcfMint = (
```